### PR TITLE
Run workload directly instead of using shell script file

### PR DESCRIPTION
**Pull request by wtrgjadmw**
- modify the way to run workload; run commands directly on bash instead of using shell script.

### CODE REVIEW

**Comment by wtrgjadmw** [comment]
delete unrelated changes🙇 I will upload them in another PR.

File: docs/servo_power_test/src/config.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ export class Config {
   private runner: TestRunner;
   private halt = true;
   private inProgress = false;
+  private configNum: number;
```
Comment (hikalium) [inline docs/servo_power_test/src/config.ts:14]: Num -> Number

File: docs/servo_power_test/src/test_runner.ts [hunk 2]
```diff
@@ -28,26 +28,18 @@ export class TestRunner {
...
-function workload () {
+  public async runWorkload(currentConfigNum: number, customScript: string) {
+    const script = `function workload () {
+  echo "current config: ${currentConfigNum}"
   ${customScript}
 }
 sleep 3
 echo "start"
-workload 1> ./test_out.log 2> ./test_err.log
+workload 1>> ./test_out.log 2>> ./test_err.log
```
Comment (hikalium) [inline docs/servo_power_test/src/test_runner.ts:38]: When will the test_{out,err}.log be deleted? Storage space on a DUT is a limited resource, so we have to care about the lifecycle of these logs. Maybe we can defer the change to the follow-up pull requests.

File: docs/servo_power_test/src/test_runner.ts [hunk 3]
```diff
@@ -28,26 +28,18 @@ export class TestRunner {
...
-    await this.dut.write('cat > ./example.sh << EOF\n');
-    await this.dut.write(btoa(script) + '\n');
-    await this.dut.write('EOF\n');
-  }
-  public async executeScript() {
-    await this.dut.write('base64 -d ./example.sh | bash\n');
-  }
-  public async executeCommand(s: string) {
-    await this.dut.write(s);
+    await this.dut.write(`echo "${btoa(script)}" | base64 -d | bash\n`);
```
Comment (hikalium) [inline docs/servo_power_test/src/test_runner.ts:42]: Yes! That's exactly what I was suggesting. nice!!!
(maybe adding "\n" at the beginning of the line improves the stability, as I described during our 1:1)

**Review by hikalium — CHANGES_REQUESTED** [review]

File: docs/servo_power_test/src/config.ts (hunk 1 above)
Comment (wtrgjadmw) [reply docs/servo_power_test/src/config.ts:14]: I fixed this in #163, so please merge the PR ahead.

File: docs/servo_power_test/src/test_runner.ts (hunk 2 above)
Comment (wtrgjadmw) [reply docs/servo_power_test/src/test_runner.ts:38]: I will fix this in another PR later🙇

Comment (wtrgjadmw) [reply docs/servo_power_test/src/test_runner.ts]: As discussed in our 1:1, I just deleted this redirect in this PR! I will add implementation to delete the files in another PR.

**Review by hikalium — APPROVED** [review]

**Comment by hikalium** [comment]
sorry for the late reply... LGTM!

**Pull request merged**